gdal/alg/gdalrasterize.cpp

```diff
@@ -54,6 +55,16 @@
 
 CPL_CVSID("$Id$")
 
+/************************************************************************/
+/*                          gvRoundBurnValue()                          */
+/************************************************************************/
+static inline
+double gvRoundBurnValue( double dfMin, double dfValue, double dfMax )
```

Comment:
This is generic code that I would think belongs in some utility file.  It has nothing to do with burn values, as such.  It could be:

```
template <class T>
const T& clamp(const T& t, const T& minimum, const T& maximum)
{
    return ((t < minimum) ? minimum : ((t > maximum) ? maximum : t));
}
```

Reply:
Yes. 
It's really not good enough here.